com.sudovanilla.penpot-desktop.yaml

```
@@ -0,0 +1,39 @@
+app-id: com.sudovanilla.penpot-desktop
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+sdk: org.freedesktop.Sdk
+rename-desktop-file: penpot-desktop.desktop
+rename-icon: penpot-desktop
+command: run.sh
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+
+modules:
+  - name: penpot-desktop
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://sudovanilla.com/distribute/applications/penpot-desktop/v0.2.33/Penpot-Desktop.AppImage
+        sha256: d448a70f45cef03a89efe2f5c28b1dae2a0a53cf627749865195bec422e7493a
+        only-arches: [x86_64]
+      - type: file
+        url: https://sudovanilla.com/distribute/applications/penpot-desktop/v0.2.33/Penpot Desktop-arm64.AppImage
+        sha256: d59c26d66344e90f8cdb58b5468adfb8b8d16364da25a643c39deff873f421c3
+        only-arches: [aarch64]
+      - type: file
+        path: ./run.sh
+      - type: file
+        path: com.sudovanilla.penpot-deskop.appdata.xml
+    build-commands:
+      - install -Dm755 ./run.sh /app/bin/run.sh
+      - chmod +x Penpot*.AppImage
+      - ./Penpot*.AppImage --appimage-extract
+      - install -D squashfs-root/usr/share/icons/hicolor/0x0/apps/penpot-desktop.png -t /app/share/icons/hicolor/512x512/apps/
+      - install -D squashfs-root/penpot-desktop.desktop -t /app/share/applications/
+      - mv squashfs-root /app/bin/penpot-desktop
+      - install -D com.sudovanilla.penpot-deskop.appdata.xml /app/share/appdata/com.sudovanilla.penpot-deskop.appdata.xml
```

```suggestion
      - install -D com.sudovanilla.penpot-deskop.appdata.xml /app/share/appdata/${FLATPAK_ID}.appdata.xml
```

given that the source filename is incorrect. At least fix that at install time.